tango_simlib/tests/test_sim_xmi_parser.py

```diff
@@ -520,12 +519,20 @@ def test_parsed_attributes(self):
         # Compare the values of the attribute properties captured in the POGO generated
         # xmi file and the ones in the parsed attribute data structure.
         for prop in expected_pressure_attr_info:
-            self.assertEquals(
-                actual_parsed_pressure_attr_info[prop],
-                expected_pressure_attr_info[prop],
-                "The expected value for the parameter '%s' does not match"
-                " with the actual value" % (prop),
-            )
+            if isinstance(actual_parsed_pressure_attr_info[prop], unicode):
```

Comment:
Not sure if `unicode` exists in Py3, this might break.
Perhaps use `try/except`